app/src/androidTest/java/ch/epfl/sdp/RegisterTest.java

```diff
@@ -46,52 +43,53 @@
     private String email;
     private String password;
     private Instrumentation.ActivityResult result;
-    private UserDataController store;
     private List<ViewAction> testCases;
     private List<Integer> testCasesInt;
     private List<Integer> emptyFields;
     private List<String> errorTexts;
-    private DependencyVisitor dv = new DependencyVisitor() {
-        @Override
-        public void setDependency(UserDataController dependency) {
-
-        }
-
-        @Override
-        public void setDependency(AuthenticationController dependency) {
-            LoginFormActivity.authenticationController = dependency;
-            RegisterFormActivity.authenticationController = dependency;
-        }
-
-        @Override
-        public void setDependency(MapApi dependency) {
-
-        }
-
-        @Override
-        public void setDependency(RemoteToSQLiteAdapter dependency) {
-
-        }
-
-        @Override
-        public void setDependency(RemoteFriendFetcher dataController) {
-
-        }
-
-        @Override
-        public void inject() {
-            setDependency(new MockAuthentication(new MockUserDataController()));
-        }
-
-    };
+//    private DependencyVisitor dv = new DependencyVisitor() {
```

Comment:
Again, don't hesitate to delete code. As the assistant told, we can take it back from github.

Reply:
I will delete it